ui/tests/drawer.spec.ts

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024-Present The UDS Authors
+
+import { expect, test } from '@playwright/test'
+
+test.describe('Drawer', async () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/workloads/pods')
+
+    await page.getByRole('cell', { name: 'podinfo-' }).click()
+  })
+
+  test.describe('is opened when clicking on a table row and', async () => {
+    test('will display Metadata details', async ({ page }) => {
+      const drawerEl = page.getByTestId('drawer')
+
+      await expect(drawerEl).toBeVisible()
+      await expect(drawerEl.getByText('Created')).toBeVisible()
+      await expect(drawerEl.getByText('Name', { exact: true })).toBeVisible()
+      await expect(drawerEl.getByText('Namespace')).toBeVisible()
```

Comment:
```suggestion
      await expect(drawerEl.getByText('Annotations')).toBeVisible()
```

"Namespace" is already visible in the table view, recommend checking for something like "Annotations", "Labels or something unique to the drawer